src/main/java/uk/gov/pay/api/utils/TrustStoreLoader.java

```diff
@@ -0,0 +1,64 @@
+package uk.gov.pay.api.utils;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.NoSuchFileException;
+import java.nio.file.Paths;
+import java.security.KeyStore;
+import java.security.KeyStoreException;
+import java.security.NoSuchAlgorithmException;
+import java.security.cert.Certificate;
+import java.security.cert.CertificateException;
+import java.security.cert.CertificateFactory;
+
+public class TrustStoreLoader {
```

Comment:
I assume this will get called during a start/restart of Dropwizard app. So this will try to reload the certs from `CERTS_PATH` during a restart even if was loaded before. If we remove the files from CERTS_PATH after a successful load we can perhaps avoid it? (unless we can be sure that trust stores handles this magically by not accepting the same cert multiple times)